### PR TITLE
Put data manipulation buttons in a menu [SATURN-1230]

### PR DESCRIPTION
**Pull request by zarsky-broad**
Verified functionality in browser, okayed by product and UX.

### CODE REVIEW

**Comment by breilly2** [comment]
Also need to run this by Customer Success in case they need to update any training/workshop materials.

File: src/pages/workspaces/workspace/Data.js [hunk 1]
```diff
@@ -625,23 +611,50 @@ class EntitiesContent extends Component {
...
-    return h(ButtonPrimary, {
-      style: { marginRight: '1rem' },
-      disabled: _.isEmpty(selectedEntities),
-      tooltip: 'Open the selected data',
-      onClick: () => this.setState({ showToolSelector: true })
-    }, [
-      'Open with...'
-    ])
+    const isSet = _.endsWith('_set', entityKey)
+    const disabled = _.isEmpty(selectedEntities)
```
Comment (panentheos) [inline src/pages/workspaces/workspace/Data.js]: If all of the items in the menu are disabled, perhaps we should disable the menu itself?

File: src/pages/workspaces/workspace/Data.js [hunk 2]
```diff
@@ -625,23 +611,50 @@ class EntitiesContent extends Component {
...
+
+    return h(PopupTrigger, {
```
Comment (panentheos) [inline src/pages/workspaces/workspace/Data.js]: This should probably have `closeOnClick` so it behaves like other menus

File: src/pages/workspaces/workspace/Data.js [hunk 3]
```diff
@@ -625,23 +611,50 @@ class EntitiesContent extends Component {
...
+          tooltip: 'Send the selected data to another workspace',
+          onClick: () => this.setState({ copyingEntities: true })
+        }, ['Export to Workspace']),
+        h(MenuButton, {
+          tooltip: 'Permanently delete the selected data',
+          disabled: disabled || Utils.editWorkspaceError(workspace),
+          onClick: () => this.setState({ deletingEntities: true })
+        }, ['Delete Data'])
+      ])
+    }, [h(Clickable, { style: { marginRight: '1rem', color: colors.accent() } }, [
```
Comment (panentheos) [inline src/pages/workspaces/workspace/Data.js]: We can use `Link` to get the color and some hover styling for free.

**Review by panentheos — APPROVED** [review]
Looks good, see a few minor UX comments.

Tested by clicking through locally.

**Pull request merged**